src/hotspot/share/opto/compile.cpp

```diff
@@ -930,6 +930,7 @@ Compile::Compile( ciEnv* ci_env,
     _log(ci_env->log()),
     _failure_reason(nullptr),
     _first_failure_details(nullptr),
+    _for_post_loop_igvn(comp_arena(), 8, 0, nullptr),
```

Comment:
Probably need to do this in a separate RFE. Looks like a few data-structures get allocated over `comp_arena` in one constructor and `ResourceArea` in the other.

Reply:
Is this still relevant?